test/Microsoft.Extensions.Configuration.EnvironmentVariables.Test/EnvironmentVariablesTest.cs

```diff
@@ -92,12 +92,12 @@ public void LoadKeyValuePairsFromAzureEnvironmentWithPrefix()
 
             envConfigSrc.Load(dict);
 
-            Assert.Equal("CustomConnStr", envConfigSrc.Get("db1:ConnectionString"));
-            Assert.Equal("SQLConnStr", envConfigSrc.Get("db2:ConnectionString"));
+            Assert.Equal("CustomConnStr", envConfigSrc.Get("db1"));
```

Comment:
I think the intent of this test was to be different from `LoadKeyValuePairsFromAzureEnvironment()` in that it would remove the "Data:" prefix because the configuration provider was initialized with that prefix. But not that we don't pre-pend "Data:" this because basically a duplicate of the previous test. 

In order to keep it a relevant test, can you please add a prefix that isn't "Data:" to the tests data in `dict`? That should also cover the scenario in which the user creates compound connection string names, e.g. "MyApplication:DefaultConnectionString".